django/db/backends/oracle/operations.py

```diff
@@ -99,7 +99,7 @@ def date_extract_sql(self, lookup_type, sql, params):
         else:
             lookup_type = lookup_type.upper()
             if not self._extract_format_re.fullmatch(lookup_type):
-                raise ValueError(f"Invalid loookup type: {lookup_type!r}")
```

Comment:
Can we add a test for this message? (in the `tests/backends/oracle/test_operations.py`)

Reply:
Sure